docs/src/main/asciidoc/security-authorization-of-web-endpoints-reference.adoc

```diff
@@ -3,9 +3,11 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
+[id="security-authorization-of-web-endpoints-reference"]
 = Authorization of Web Endpoints
 
```

Comment:
Remove blank line here.

Reply:
I believe there might be a tooling reason for preserving these blank lines, which are present in many/most of our files. For now, I'll leave this alone. 
If we need to remove the blank lines at some point, we'll do it systematically across the whole doc set.